adapters/tlsio_openssl.c

```diff
@@ -106,6 +106,7 @@ static void* tlsio_openssl_CloneOption(const char* name, const void* value)
     {
         if (strcmp(name, OPTION_UNDERLYING_IO_OPTIONS) == 0)
         {
+            //result = (void*)OptionHandler_Clone((OPTIONHANDLER_HANDLE)value);
```

Comment:
what's the purpose of this guy here?

Reply:
To check if someone would catch it.

Reply:
![gif](https://media.giphy.com/media/3o7bu57lYhUEFiYDSM/giphy.gif)

Reply:
Fixed.